packages/gatsby/src/schema/print.ts

```diff
@@ -1,17 +1,44 @@
-const fs = require(`fs-extra`)
-const {
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
My guess is we need this because we have anonymous functions as consts and they don't hoist.

Reply:
There's lot of helper functions and recursive calls happening. I started re-arranging this file to satisfy the rule, but then the diff looked even more horrible. I'm not sure if it's worth doing that

Reply:
Adjusted in [`fa9cc10` (#34787)](https://github.com/gatsbyjs/gatsby/pull/34787/commits/fa9cc10a1d350887e60e416cb41770f3685d6a3b)